Client/Frontend/Library/Bookmarks/BookmarksPanel.swift

```diff
@@ -137,13 +137,16 @@ class BookmarksPanel: SiteTableViewController, LibraryPanel, CanRemoveQuickActio
             }
 
             // Ecosia: add empty state
```

Comment:
Is this tag still relevant after removing the code that was after it?

Reply:
It kind of still is as the code just moved to the method. So I'd leave it for now.